book/src/06_system_data.md

```diff
@@ -102,3 +102,29 @@ There are actually three ways of doing this:
   that the order here is important, using `Read<'a, Option<Resource>>` will not 
   result in the same behavior (it will try to fetch `Option<Resource>` from `World`, 
   instead of doing an optional check if `Resource` exists).
+
+## Specifying `SystemData`
+
+As mentioned earlier, `SystemData` is implemented for tuples of up to 26 elements. Should you ever need
+more, you could even nest these tuples. However, at some point you can't keep track of all the elements anymore.
+That's why you can also create your own `SystemData` bundle using a struct:
```

Comment:
Btw what happens if you try to derive it for enum? I realised you could try to do this after reading this.

Reply:
Doesn't work, which variant should it select?

Reply:
Guessed as much :D